core/amber/requirements.txt

```diff
@@ -25,4 +25,5 @@ python-lsp-server[websockets]
 bidict==0.22.0
 cached_property
 psutil
-transformers
\ No newline at end of file
+transformers
+torch
```

Comment:
please move those into `operator-requirements.txt`. They are not for core engine.